docs/data/sensitive-data/sd_desktop.md

```diff
@@ -29,7 +29,7 @@ In the following user guide, you can learn how to:
 
 ## Before you start
 
-* All the members belonging to a specific CSC project can access the same computing virtual Desktop. Currently, it is possible to launch 3 virtual Desktops (or computing environment) for each CSC project. Each CSC project has its private Desktop, and each Desktop is isolated from other CSC projects or CSC accounts unless you authorize it.
+* All the members belonging to a specific CSC project can access the same computing virtual Desktops. Currently, it is possible to launch 3 virtual Desktops (or computing environment) for each CSC project. Each CSC project has its private Desktop, and each Desktop is isolated from other CSC projects or CSC accounts unless you authorize it.
```

Comment:
Is computing needed in "computing virtual Desktops", and does it really make sense to capitalize "Desktop" when you in general talk about the desktops (and not SD Desktop brand name specifically)